src/docker-utils.coffee

```diff
@@ -15,6 +17,72 @@ Promise.promisifyAll(docker.getContainer().constructor.prototype)
 
 exports.docker = docker
 
+# Create an array of (repoTag, image_id, created) tuples like the output of `docker images`
+listRepoTagsAsync = ->
+	docker.listImagesAsync()
+	.then (images) ->
+		images = _.sortByOrder(images, 'Created', [ false ])
+		ret = []
+		for image in images
```

Comment:
This can also be:
```coffee
_(images)
.sortByOrder('Created', [ false ])
.map (image) ->
	for repoTag in image.RepoTags
		[ repoTag, image.Id, image.Created ]
.flatten()
.value()
```